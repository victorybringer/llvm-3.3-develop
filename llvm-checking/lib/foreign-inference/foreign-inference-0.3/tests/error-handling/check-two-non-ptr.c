#include <stdlib.h>
#include <unistd.h>

extern int *g1;
extern int *g2;

int target(int fd) {
  char buffer[100];
  int rc1 = read(fd, buffer, 1);
  int rc2 = read(fd, buffer, 2);

  if(rc1 == -1 || rc2 == -1)
    return -30;

  *g1 = rc1;
  *g2 = rc2;

  return 0;
}
