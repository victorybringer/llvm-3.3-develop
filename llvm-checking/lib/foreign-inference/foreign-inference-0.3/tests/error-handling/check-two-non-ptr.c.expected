fromList [ ("target", (fromList [], ReturnConstantInt (fromList [-30]))) ]
